src/languages/es.js

```diff
@@ -586,6 +586,14 @@ export default {
         listOfRestrictedBusinesses: 'lista de negocios restringidos',
         incorporationDatePlaceholder: 'Fecha de inicio (aaaa-mm-dd)',
         companyPhonePlaceholder: '(prefijo) + (número)',
+        incorporationTypes: {
+            LLC: 'LLC',
+            CORPORATION: 'Corp',
+            PARTNERSHIP: 'Camaradería',
```

Comment:
This does not sound like the correct word, I have no idea what this partnership type of company is or how is it called in spanish... let me post this on slack and get back to you

Reply:
Let's use `Sociedad`

Reply:
Corrected and pushed new commit. Thanks.